test.html

```diff
@@ -0,0 +1,126 @@
+<html>
+<style>
+
+.container {
+  padding-right: 15px;

```

Comment:
Generally padding is added in rows. Container should have rows and rows inturn columns. Not directly